c/crossmemory.c

```diff
@@ -2808,12 +2808,13 @@ static void printDisplayConfigCommandResponse(CrossMemoryServer *server, CMSWTOR
   int consoleID = routeInfo->consoleID;
   CrossMemoryServerGlobalArea *globalArea = server->globalArea;
 
-  zowelog(NULL, LOG_COMP_ID_CMS, ZOWE_LOG_INFO,
 #ifdef CROSS_MEMORY_SERVER_DEBUG
-      CMS_LOG_DISP_CMD_RESULT_MSG"Server name - \'%16.16s\' (debug mode)\n"
+  char* formatStringPrefix = CMS_LOG_DISP_CMD_RESULT_MSG"Server name - \'%16.16s\' (debug mode)\n";
 #else
-      CMS_LOG_DISP_CMD_RESULT_MSG"Server name - \'%16.16s\'\n"
+  char* formatStringPrefix = CMS_LOG_DISP_CMD_RESULT_MSG"Server name - \'%16.16s\'\n";
 #endif
+
+  zowelog(NULL, LOG_COMP_ID_CMS, ZOWE_LOG_INFO, strcat(formatStringPrefix,
```

Comment:
Do you `strcat` into a string literal? Have you tested this code? I would expect an S0C4.

Reply:
Agreed! it's a buffer overflow into the literal pool! `strcat` is a code smell and should never be used.